pkg/virt-handler/vm_test.go

```diff
@@ -2239,6 +2240,33 @@ var _ = Describe("VirtualMachineInstance", func() {
 			Expect(condition.Reason).To(Equal(v1.VirtualMachineInstanceReasonVirtIOFSNotMigratable))
 		})
 
+		It("should not be allowed to live-migrate if the VMI does use masquerade to connect to the pod network", func() {
+			vmi := api2.NewMinimalVMI("testvmi")
+			interfaceName := "interface_name"
+
+			strategy := v1.EvictionStrategyLiveMigrate
+			vmi.Spec.EvictionStrategy = &strategy
+
+			vmi.Spec.Networks = []v1.Network{
+				{
+					Name:          interfaceName,
+					NetworkSource: v1.NetworkSource{Pod: &v1.PodNetwork{}},
+				},
+			}
+			vmi.Spec.Domain.Devices.Interfaces = []v1.Interface{
+				{
+					Name: interfaceName,
+					InterfaceBindingMethod: v1.InterfaceBindingMethod{
+						Bridge: &v1.InterfaceBridge{},
+					},
+				},
+			}
```

Comment:
Just a nit: this can be replaced with test functions:
```
vmi.Spec.Domain.Devices.Interfaces = []v1.Interface{*v1.DefaultBridgeNetworkInterface()}
vmi.Spec.Networks = []v1.Network{*v1.DefaultPodNetwork()}
```

Reply:
Fixed, thanks.